### DISCUSSION TITLE
Cypher support for date() function

### DISCUSSION THREAD

**Discussion by CAQ** [TuGraph-family/tugraph-db#91] [Feature Request]
1. RETURN datetime("2022-10-18 00:00:00") AS a    (Success)
2. RETURN datetime("2022-10-18") AS a
- failed to parse string 2022-10-18 into DateTime
3. RETURN date("2022-10-18") AS a
- unregistered function: date

Did I make a mistake on Cypher syntax, or please support the date() function. Thanks.

**Comment by wangtao9** [comment]
The `date()` function is not implemented yet, we will implement it later. @spasserby
